sdk2refactor/pom.xml

```diff
@@ -0,0 +1,284 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
So we're building out a new maven structure in a subdirectory, and we'll move the source code in there later?

I'd much prefer using a branch for this instead. Why don't we create a sdk2 branch on the main github repo, do our work there, and merge back into master when ready? This way we can put all the new POM files in their final locations from the start. This would also help with comparing with what we already have.

Reply:
I'm reluctant to do all development on a branch, since the final changes will be pretty large. It would be much nicer if we could merge this into master slowly, instead of all at once. Would you be open to me restructuring the the SDKv1 code underneath the SDKv1 module now? Everything would be in its final spot, aside from README's and examples.